report_aeroo/static/src/js/report/reportactionmanager.js

```diff
@@ -18,12 +18,17 @@ ActionManager.include({
         if (c_action.report_type !== 'aeroo') {
             return self._super(action, options);
         }
-        
+
         framework.blockUI();
         var aeroo_url = 'report/aeroo/' + c_action.report_name;
-        if(c_action.context.active_ids){
-            aeroo_url += '/' + c_action.context.active_ids.join(',');
+        if (_.isUndefined(action.data) || _.isNull(action.data) || (_.isObject(action.data) && _.isEmpty(action.data))) {
```

Comment:
This is more similar as odoo does. Without this changes context and data where lost and not reaching the controller